src/mysql/create.sql

```diff
@@ -14,7 +14,9 @@ CREATE TABLE Major (
 
 CREATE TABLE User (
   id INT NOT NULL AUTO_INCREMENT,
-  name VARCHAR(255),
+  fname VARCHAR(255),
+  lname VARCHAR(255),
+  username VARCHAR(255),
```

Comment:
If you update the tables here, make sure you also update the test tables!

Reply:
Done! (I did notice this in another branch, but got it fixed here as well.)